duckduckgo/iOS
# Check metadata length

Task/Issue URL: https://app.asana.com/0/414709148257752/1203659089823819/f

**Description**:
Since App Store metadata fields have strict length requirements and they are only checked   A script to check if metadata fields are of appropriate length. The script can be run without parameter to check the metadata under default path or with a parameter for an external folder.

**Steps to test this PR**:
1. Run `check_metadata_length.sh` inside repo (run without parameter)
2. Modify some of the files under `/fastlane/metadata` to exceed the limits and rerun the `check_metadata_length.sh` script to see if it detects errors.
3. Open the project in Xcode and start a build - the same errors should be displayed in Xcode and should fail the build.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
